From 81f37daa1443cb5df426eeff72721213096310ca Mon Sep 17 00:00:00 2001
From: Mingcong Bai <baimingcong@uniontech.com>
Date: Thu, 27 Jun 2024 14:37:53 +0800
Subject: [PATCH 027/100] net: stmmac: fix dwmac-phytium build on 6.9

Declare phytium_dwmac_remove() as a static function to resolve a missing
prototype warning.

Signed-off-by: Mingcong Bai <baimingcong@uniontech.com>
Ref: https://github.com/deepin-community/kernel-rolling/commit/ea0ab78657ab6bbb4717b176d4c802a4a032b710
---
 drivers/net/ethernet/stmicro/stmmac/dwmac-phytium.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/net/ethernet/stmicro/stmmac/dwmac-phytium.c b/drivers/net/ethernet/stmicro/stmmac/dwmac-phytium.c
index 6e8e44730bec6..6ecfa671b7f37 100644
--- a/drivers/net/ethernet/stmicro/stmmac/dwmac-phytium.c
+++ b/drivers/net/ethernet/stmicro/stmmac/dwmac-phytium.c
@@ -176,7 +176,7 @@ static int phytium_dwmac_probe(struct platform_device *pdev)
 	return  stmmac_dvr_probe(&pdev->dev, plat, &stmmac_res);
 }
 
-int phytium_dwmac_remove(struct platform_device *pdev)
+static int phytium_dwmac_remove(struct platform_device *pdev)
 {
 	struct net_device *ndev = platform_get_drvdata(pdev);
 	struct stmmac_priv *priv = netdev_priv(ndev);
-- 
2.46.0

